build-system/sauce_connect/start_sauce_connect.sh

```diff
@@ -31,6 +35,24 @@ READY_FILE="sauce_connect_ready"
 READY_DELAY_SECS=60
 LOG_PREFIX=$(YELLOW "start_sauce_connect.sh")
 
+# Check the status of the Sauce Labs service
+echo "$LOG_PREFIX Fetching current Sauce Labs service status from $(CYAN "$STATUS_URL")..."
+SAUCE_STATUS="$(curl -s "$AUTHENTICATED_STATUS_URL")"
```

Comment:
What happens if `curl` returns a non-zero exit status? Does the script move on or does it crash?
(I tested, and the returned HTTP code doesn't change the exit code, but there could be other reasons why `curl` could fail - e.g., if the domain name cannot be resolved for whatever reason)

Reply:
Good point. Just checked. The script moves on after printing the error message from curl. This should be sufficient for our purpose.